app/views/layouts/admin.html.erb

```diff
@@ -26,6 +26,7 @@
         <li class="<%= 'active' if current_page?(controller: '/admin/firms') %>"><%= link_to 'Firms', admin_firms_path %></li>
         <li class="<%= 'active' if current_page?(controller: '/admin/advisers') %>"><%= link_to 'Advisers', admin_advisers_path %></li>
         <li class="<%= 'active' if current_page?(controller: '/admin/principals') %>"><%= link_to 'Principals', admin_principals_path %></li>
+        <li class="<%= 'active' if current_page?(controller: '/admin/metrics') %>"><%= link_to 'Metrics', admin_metrics_path %></li>
```

Comment:
I think I'll probably just replace this with https://github.com/comfy/active_link_to as it solves the problem nicely.